subsys/shell/modules/kernel_service.c

```diff
@@ -73,9 +74,19 @@ static void shell_tdata_dump(const struct k_thread *thread, void *user_data)
 		      thread,
 		      tname ? tname : "NA");
 	shell_fprintf((const struct shell *)user_data, SHELL_NORMAL,
-		      "\toptions: 0x%x, priority: %d\n",
+		      "\toptions: 0x%x, priority: %d timeout %d\n",
```

Comment:
`timeout:`

Reply:
Oups. Thank you for noticing this typo. Done